packages/data-flow/README.md

```diff
@@ -90,9 +90,14 @@ The `EventsProcessor` class is responsible for processing events in the processi
 
 The `EventsFetcher` class is responsible for fetching events from the blockchain.
 
-### [StrategyRegistry](./src/strategyRegistry.ts)
+### [StrategyRegistry](./src/registries/)
 
 The `StrategyRegistry` stores strategy IDs to populate strategy events with them given the Strategy address.
+There are 3 implementations:
+
+-   `InMemoryStrategyRegistry`: stores map in-memory
```

Comment:
maybe we should get rid of this one since it will be no longer used and adds noise here

Reply:
okii makes sense

Reply:
fixed: [aa3ee27](https://github.com/defi-wonderland/grants-stack-indexer-v2/pull/40/commits/aa3ee2776048258d2dccdc6b2be57f5dcdc5b8c1)